### PR TITLE
Fix match fields masks (Issue #75)

### PR DESCRIPTION
**Pull request by keniachang**
Implement mask for supported match fields in OpenFlow 1.3

### CODE REVIEW

File: v0x04/match_fields.py [hunk 1]
```diff
@@ -13,6 +13,30 @@
...
+    strbits = strbits[2:]
+    netmask = 0
+    for i in range(size):
+        if strbits[i] == '1':
+            netmask += 1
+        else:
+            break
+    return netmask
+
+
```
Comment (ajoaoff) [inline v0x04/match_fields.py:39]: There is a `utils.py` module in `v0x04` directory. These auxiliary functions should be there.

File: v0x04/match_fields.py [hunk 2]
```diff
@@ -168,16 +252,23 @@ class MatchNwSrc(MatchField):
...
+                      oxm_value=value_bytes)
 
     @classmethod
     def from_of_tlv(cls, tlv):
         """Return an instance from a pyof OXM TLV."""
         ip_address = IPAddress()
         ip_address.unpack(tlv.oxm_value)
-        ip_str = str(ip_address)
-        return cls(ip_str)
+        value = str(ip_address)
```
Comment (ajoaoff) [inline v0x04/match_fields.py:268]: In the mac address fields you first attributed the value of `str(hw_address)` to a `addr_str` variable, and then attributed it to `value`. Here, you are attributing `str(ip_address)` directly into `value`. You should do both attributions the same way.

File: v0x04/match_fields.py (hunk 1 above)
Comment (keniachang) [reply v0x04/match_fields.py]: Hi, it seems like moving these functions to utils.py and import them from utils are causing the updated commit to fail the Scrutinizer check.

**Comment by keniachang** [comment]
Thank you for the suggestions, I have updated the files.

**Pull request merged**